backend/README.md

```diff
@@ -9,6 +9,10 @@ Welcome to the Penn Courses Backend (PCX)!
 
 ### Running the Backend with Docker-Compose
```

Comment:
I think we should move the local dev environment thing to the top, and move down `Running the backend with Docker-Compose`.